CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Continued to redesign the user interface: this time the editor got a fresh coat of paint:
   - The buttons were changed to icons.
   - Removed the hidden feature that a double click in the editor inserted the current date.
+- All authors and editors are separated using semicolons when exporting to csv.
```

Comment:
You can leave it there for the moment. Before a new version release we usually check and reorder the changelog